tests/lib.rs

```diff
@@ -58,6 +58,7 @@ fn test_basic() {
         (Base32Z, "hxf1zgedpcfzg1ebb"),
         (Base36Lower, "k2lcpzo5yikidynfl"),
         (Base36Upper, "K2LCPZO5YIKIDYNFL"),
+        (Base45, "RRFF.OEB$D5/DZ24"),
```

Comment:
Please also add base45 to the other tests. Parts of the fixtures can be found at https://github.com/multiformats/multibase/pull/125/files.